core/src/main/scala/com/github/mjakubowski84/parquet4s/ParquetWriter.scala

```diff
@@ -29,21 +31,41 @@ trait ParquetWriter[T] {
     */
   def write(path: String, data: Iterable[T], options: ParquetWriter.Options)
 
+  /**
+    * Writes data out to given OutputFile
+    * @param outputFile a stream representation
+    * @param data data to write
+    * @param options configuration of how Parquet files should be created and written
+    */
+  def writeS(outputFile: OutputFile, data: Iterable[T], options: ParquetWriter.Options): Unit
```

Comment:
There should be a util to write to just `java.io.File`. `OutputFile` interface should be private.